src/util/helpers.ts

```diff
@@ -28,6 +29,15 @@ export const formatDec = (num: number) => {
   return Number(num.toFixed(Config.DECIMAL)).toString();
 };
 
+export const toBitcoinAmount = (amount: number | string) => {
```

Comment:
you can type the function return so we know what to expect: 
```suggestion
export const toBitcoinAmount = (amount: number | string): string => {
```